src/common/Experiments/Experiments-Live/Experiments.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Experimentation
+{
+    using System.Text.Json;
+    using Microsoft.PowerToys.Settings.UI.Library.Telemetry.Events;
+    using Microsoft.PowerToys.Telemetry;
+    using Microsoft.VariantAssignment.Client;
+    using Microsoft.VariantAssignment.Contract;
+
+    public class Experiments
+    {
+        private enum FeatureFlag
+        {
+            Current,
+            Alternate,
+        }
+
+        public bool EnableLandingPageExperiment()
+        {
+            Experiments varServ = new Experiments();
+            varServ.VariantAssignmentProvider_Initialize();
+            var landingPageExperiment = varServ.IsExperiment;
+
+            return landingPageExperiment;
+        }
+
+        private void VariantAssignmentProvider_Initialize()
+        {
+            var vaSettings = new VariantAssignmentClientSettings
+            {
+                Endpoint = new Uri("https://default.exp-tas.com/exptas77/a7a397e7-6fbe-4f21-a4e9-3f542e4b000e-exppowertoys/api/v1/tas"),
+                EnableCaching = true,
+                ResponseCacheTime = TimeSpan.FromMinutes(5),
+            };
+
+            var vaClient = vaSettings.GetTreatmentAssignmentServiceClient();
+            var vaRequest = GetVariantAssignmentRequest();
+
+            try
+            {
+                var task = vaClient.GetVariantAssignmentsAsync(vaRequest);
+                var result = task.Result;
```

Comment:
Accessing the result of an async operation will block until the operation completes. Since this is on applciation startup, should there be a timeout here?

Reply:
Added a 500 ms timeout!